src/main/scala/no/gdl/bookapi/controller/CategoriesController.scala

```diff
@@ -0,0 +1,46 @@
+/*
+ * Part of GDL book_api.
+ * Copyright (C) 2018 Global Digital Library
+ *
+ * See LICENSE
+ */
+
+package no.gdl.bookapi.controller
+
+import io.digitallibrary.language.model.LanguageTag
+import no.gdl.bookapi.BookApiProperties
+import no.gdl.bookapi.model.api
+import no.gdl.bookapi.service.ReadService
+import org.scalatra.swagger.{ResponseMessage, Swagger, SwaggerSupport}
+
+
+trait CategoriesController {
+  this: ReadService =>
+  val categoriesController: CategoriesController
+
+  class CategoriesController(implicit val swagger: Swagger) extends GdlController with SwaggerSupport {
+    protected val applicationDescription = "API for getting a hierarchy of categories and reading levels of published books."
+
+    val defaultLanguage = LanguageTag(BookApiProperties.DefaultLanguage)
+
+    registerModel[api.Error]
+    val response500 = ResponseMessage(500, "Unknown error", Some("Error"))
+
+    private val getCategoriesForLanguage = (apiOperation[Map[String, Seq[String]]]("getCategoriesForLanguage")
```

Comment:
Swagger-dokumentasjonen for dette endepunktet blir ganske vanskelig å forstå med denne retur-typen. Hadde det vært fornuftig å lage en case-class som vi kan dokumentere litt bedre?